lib/plugins/laser.rb

```diff
@@ -0,0 +1,12 @@
+# :laser:
+#   :rules:
+#     - :length:
+#       - 16
+#       - 17
+#       - 18
+#       - 19
+#      :prefixes:
+#        - '6304'
+#        - '6706'
+#        - '6771'
+CreditCardValidations.add_brand(:laser, {length: [16, 17, 18, 19], prefixes: %w(6304 6706 6771)})
```

Comment:
Line is too long. [97/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Final newline missing.